R/BiocManager-pkg.R

```diff
@@ -13,25 +13,25 @@ NULL
 #' Main functions are as follows; additional help is available for
 #' each function, e.g., `?BiocManager::version`.
 #'
-#' - `BiocManager::install()`: Install or update packages from
+#' - [`BiocManager::install()`]: Install or update packages from
```

Comment:
These are meant to be formatted as code giving the fully resolved command `BiocManager::install()`. The update formats this as `install()`. The reason for using the fully resolved command is that represents 'best practice' -- no ambiguity about which package `install()` comes from. 

Also, it seemed to me that it was more helpful to have the code-formatted fully resolved command, than to aim for the fully formatted command AND link, which would render in a confusing way.

This comment applies throughout.